cmd/builder/internal/builder/config.go

```diff
@@ -88,8 +88,8 @@ func NewDefaultConfig() Config {
 	}
 }
 
-// Validate checks whether the current configuration is valid
-func (c *Config) Validate() error {
+// ValidateAndSetGoPath checks whether the current configuration is valid and sets go path
+func (c *Config) ValidateAndSetGoPath() error {
```

Comment:
I think I would prefer to split this function into two:

- SetGoPath
- Validate

The current callers of `Validate` will then need to call both.

Reply:
Done @jpkrohling